skops/card/_model_card.py

```diff
@@ -373,6 +374,47 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_permutation_importances(
+        self, feature_importances, columns, plot_file, plot_name
```

Comment:
I think you forgot to set the default argument for `plot_file` and `plot_name`, which is why lines 414 and 415 are not triggered. However, instead of setting them to `None`, I would just put the actual values, i.e. `plot_file="feature_importances.png"` and `plot_name="Permutation importances"`.